richardltyler/ideabox-boilerplate
# Fix file names in html sources

PR 
## Pull Request

<br>

### Description

##### What does this PR do?

* move js files to src directory
* fixes file names in html sources

##### What does it fix?

* organization

##### Is this a feature or a fix?

* fix

##### Where should the reviewer start?

* GUI

##### How should this be tested?
 
* check that file names in html match the names/map of files
* check location of js files. should be in src directory

<br>

***
